.github/workflows/build.yaml

```diff
@@ -77,7 +77,7 @@ jobs:
           ENVIRONMENT=test
           if [[ $GITHUB_REF == refs/tags/* ]]; then
             VERSION=${GITHUB_REF#refs/tags/}
-            ENVIRONMENT=prod
+            ENVIRONMENT=production
```

Comment:
by default is already `production`, do you need to set it anyway?

Reply:
It defaults to `test` and only when building a tag it sets to `production`.